wiki/Beatmap_Editor/id.md

```diff
@@ -1,22 +1,42 @@
----
-outdated: true
----
+# Beatmap editor
 
-# Penyunting Beatmap
+![Sebuah screenshot dari beatmap editor pada bagian compose tab memperlihatkan beatmap milik phisifat](img/compose.jpg)
 
-![osu!Editor Windowed mode](Compose/img/Compose.jpg "osu!Editor Windowed mode")
+*Untuk tombol pintas beatmap editor, silahkan lihat: [Referensi Tombol Pintasan § Beatmap editor](/wiki/Shortcut_key_reference#beatmap-editor)*
 
-**Penyunting beatmap** (beatmap editor) adalah osu! in-game graphical interface untuk membuat [Beatmaps](/wiki/Beatmaps). Hal ini dipisahkan menjadi 4 modes:
+**Beatmap Editor** adalah grafis antarmuka osu! yang digunakan untuk membuat [beatmaps](/wiki/Beatmaps). [Beatmapping](/wiki/Beatmapping) adalah sebutan dari proses pembuatan beatmap untuk osu!.
 
-- [Compose](Compose): Dimana [Hit Objects](/wiki/Hit_Objects) di letakkan dan di edit. Sebagian besar waktu didalam editor adalah disini.
-- [Design](Design): Tempat dimana untuk membuat Storyboard, kebanyakan mappers lebih menggunakan "Background Image/Video" button disini.
-- [Timing](Timing): Tempat dimana untuk mengatur BPM dan offset. Disini juga untuk mengatur beberapa Hitsound.
-- [Song Setup](Song_Setup): Di sinilah berbagai pilihan tentang Maps yang akan ditetapkan. Dan termasuk combo colors dan juga difficulty settings.
+Beatmap Editor terbagi menjadi empat bagian, yang dimana bisa dilihat di atas kiri pojok pada editor:
 
-## Artikel Terkait
+- Compose
+- Design
+- Timing
+- Song setup
 
-- [Beatmapping](/wiki/Beatmapping) (Basic Mapping)
-- [Beat Snap Divisor](Beat_Snap_Divisor)
-- [Distance Snap](Distance_Snap)
-- [Custom Sample Overrides](/wiki/Guides/Custom_Sample_Overrides)
-- [Kiai Time](Kiai_Time)
+## Compose
+
+[Compose](/wiki/Beatmap_Editor/Compose) adalah dimana semua [hit objects](/wiki/Hit_Objects) dan elemen lainnya yang termasuk dalam gameplay dibuat. Kebanyakan mappers meluangkan waktunya di dalam tab ini dibandingkan dengan yang lain.
+
+## Design
+
+[Design](/wiki/Beatmap_Editor/Design) tab menawarkan cara untuk membuat [storyboards](/wiki/Storyboards), efek visual untuk sebuah beatmaps. Karena storyboards memiliki efek yang kompleks dan memerlukan banyak perintah storyboards, mappers juga menggunakan [storyboard scripting](/wiki/Storyboard_Scripting) langsung tanpa menggunakan design tab.
+
+## Timing
+
+[Timing](/wiki/Beatmap_Editor/Timing) digunakan untuk mengatur timing sections dalam sebuah musik dan mengontrol hitsound beatmap. Ini sangat penting semenjak beatmap membutuhkan timing yang sesuai dan umpan balik yang tepat untuk pengalaman gameplay yang sesuai dengan ritme musik.
+
+## Song setup
+
+[Song setup](/wiki/Beatmap_Editor/Song_Setup) digunakan untuk mengisi [metadata](/wiki/Beatmap_Editor/Song_Setup#song-and-map-metadata) dari sebuah beatmap dan hal-hal universal untuk [beatmapset](/wiki/Beatmaps/Beatmapsets). Ini sudah tersedia dengan judul lagu, artis, nama dari tingkat kesulitan, wrna kombo, dan lain-lain.
```

Comment:
Tab [Song Setup](/wiki/Beatmap_Editor/Song_Setup) merupakan bagian dari editor yang dipergunakan untuk mengatur informasi [metadata](/wiki/Beatmap_Editor/Song_Setup#song-and-map-metadata) dan seluk-beluk lainnya yang berlaku secara umum pada [beatmapset](/wiki/Beatmaps/Beatmapsets) yang bersangkutan. Hal ini meliputi antara lain judul lagu, nama artis, nama tingkat kesultian, warna kombo, dan masih banyak lagi.